core/types/block.go

```diff
@@ -195,7 +195,7 @@ func (h *Header) EmptyReceipts() bool {
 
 // EmptyWithdrawalsHash returns true if there are no WithdrawalsHash for this header/block.
```

Comment:
Maybe we should change the description of this function to make it more clear.

Reply:
> Maybe we should change the description of this function to make it more clear.

ok